src/DurableTask.Netherite/DurableTask.Netherite.csproj

```diff
@@ -40,10 +40,10 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.33.0" />
+    <PackageReference Include="Azure.Core" Version="1.38.0" />
     <PackageReference Include="Azure.Data.Tables" Version="12.8.0" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.9.2" />
-    <PackageReference Include="Microsoft.Azure.EventHubs.Processor" Version="4.3.2" />
+		<PackageReference Include="Azure.Messaging.EventHubs" Version="5.11.2" />
+		<PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.11.2" />
```

Comment:
Looks right on my screen (for me in VS tabs appear as 2 spaces in csproj files) but not for someone where tabs are displayed as 4 spaces.

Ideally I would want VS to replace all tabs with spaces (it does this in C# already) but I suppose it would need some kind of configuration.